### PR TITLE
Feature/ode 68 footer for blog

### PR DESCRIPTION
**Pull request by nguyenvl179**
Review footer for blog.

### CODE REVIEW

**Comment by ngvuthanhnhan** [comment]
Screen Shot 2020-05-29 at 9 27 21 PM
Khi width = 780 đến 1110 thì nút **Subcribe** bị mất chữ nhe, feature hay bug đây :((

File: resources/css/index.css [hunk 1]
```diff
@@ -140,8 +140,97 @@
 }
 
 /* Footer */
-#footer {
-  height: 256px;
-  background-color: azure;
+#footer{
```
[the remark this answers is not in the document]
Comment (nguyenvl179) [reply resources/css/index.css:143]: I fixed.

File: index.html [hunk 2]
```diff
@@ -47,9 +47,37 @@ <h1>About</h1>
...
+        </div>
+        <div class="categories">
+            <h3 class="title">Categories</h3>
+            <div class="contain-box">
+                <a href="#" class="box">Custom</a>
+                <a href="#" class="box">Me</a>
+                <a href="#" class="box">Contact</a>
+                <a href="#" class="box">Custom</a>
+                <a href="#" class="box">Me</a>
+                <a href="#" class="box">Contact</a>
```
[the remark this answers is not in the document]
Comment (ngoc-nguyenminh) [reply index.html:68]: Try to modify the category so that you don't have to write classes on html :+1: 
Write dummy data and give a try on js file :))))

**Comment by nguyenvl179** [comment]
> Screen Shot 2020-05-29 at 9 27 21 PM
> 
> Khi width = 780 đến 1110 thì nút **Subscribe** bị mất chữ nhe, feature hay bug đây :((
Mình sửa r :)))).

**Pull request merged**